src/yAxis.js

```diff
@@ -46,7 +47,11 @@ export default class YAxis extends Component<void, any, any> {
 		}
 		minBound = (minBound < 0) ? 0 : minBound;
 		let label = minBound + (maxBound - minBound) / this.props.verticalGridStep * index;
-		label = Math.round(label);
+
+		if (this.props.yAxisUseDecimal) {
+			label = Math.round(label);
+		}
```

Comment:
Wouldn't it be the inverse? i.e. `if (!this.props.yAxisUseDecimal) label = Math.round(label)` ?